lib/update/cluster/automatic.go

```diff
@@ -132,3 +132,14 @@ func ShutdownClusterAgents(ctx context.Context, remote rpc.AgentRepository) erro
 	err = rpc.ShutdownAgents(ctx, nodes, log.StandardLogger(), remote)
 	return trace.Wrap(err)
 }
+
+// AgentStatus requests the statuses of all agents
+func AgentStatus(ctx context.Context, remote rpc.AgentRepository) ([]rpc.GravityAgentStatus, error) {
+	nodes, err := kubectl.GetNodesAddr(ctx)
```

Comment:
I think we should retrieve the nodes from the cluster state, not k8s. If you pass an operator here, you can use it:

```go
// in cli/rpcagent.go
func rpcAgentStatus(env *localenv.LocalEnvironment) error {
  operator, err := env.ClusterOperator()
  ...
  cluster.AgentStatus(ctx, operator, remote)
}

// here
func AgentStatus(ctx context.Context, operator ops.Operator, remote ...) () {
  cluster, err := operator.GetLocalSite()
  ...
  cluster.ClusterState.Servers // <-- cluster nodes
}
```